CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.5.0] 2020-02-03
+## [0.5.1] 2020-02-03
```

Comment:
Diff looks strange. I made a poo poo in my previous PR. If you check at the bottom there is second 0.5.0 release listed.